### PR TITLE
Feature/map box

### PR DESCRIPTION
**Pull request by gauthierboeshertz**

### CODE REVIEW

**Comment by gauthierboeshertz** [comment]
I can't resolve the -0.7 code coverage, it's about functions located at the bottom of the MapActivity file that I can't test

**Review by Warhedgehog — APPROVED** [review]
Looks good to me

File: app/src/androidTest/java/sdp/moneyrun/MapInstrumentedTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+public class MapInstrumentedTest {
+
+        @Test
+        public void AddMarkerAddsMarker() {
+            try (ActivityScenario<MapActivity> scenario = ActivityScenario.launch(MapActivity.class)) {
+                float lat = 12f;
+                float lon = 12f;
+                try {
+                    Thread.sleep(2500);
```
Comment (Tesa27) [inline app/src/androidTest/java/sdp/moneyrun/MapInstrumentedTest.java:19]: Why do we need to wait here?

Comment (gauthierboeshertz) [reply app/src/androidTest/java/sdp/moneyrun/MapInstrumentedTest.java:19]: Because the map takes time to show and does asynchronous stuff.

**Pull request merged**